bartop-ui/src/components/Auth/Failure/Failure.stories.js

```diff
@@ -1,11 +0,0 @@
-import React from 'react';
```

Comment:
I don't think that `stories` for full view components are that useful (yet). I like them for the visual components, like Button, TextInput, etc